.gitignore

```diff
@@ -1,3 +1,4 @@
 bower_components/
 *.log
 node_modules/
+.idea
```

Comment:
I prefer having editor- and setup-specific files as part of a global ignore. I love everything else though. :)